test/withTranslation.spec.js

```diff
@@ -23,15 +23,21 @@ describe('withTranslation', () => {
 
   it('should render correct content', () => {
     const HocElement = withTranslation()(TestComponent);
-    const wrapper = mount(<HocElement />);
-    // console.log(wrapper.debug());
-    expect(wrapper.contains(<div>test</div>)).toBe(true);
+    const { container } = render(<HocElement />);
+    // screen.debug();
+    expect(container).toMatchInlineSnapshot(`
```

Comment:
[minor] @tigerabrodi other files use `expect(container.firstChild).toMatchInlineSnapshot`, might be good to keep it the same here for consistency

Reply:
Oh you were fast :two_hearts: :smiling_face_with_three_hearts:, defo, I just changed it once I noticed that, regarding that which I wrote in Discord, I had not seen this comment from you yet :confused:, you are awesome :smiley: